dfencoder/autoencoder.py

```diff
@@ -1010,25 +1018,33 @@ def get_results(self, df, return_abs=False):
 
         combined_loss = torch.cat([mse_scaled, bce_scaled, cce_scaled], dim=1)
 
+        if self.loss_scaler_str == 'standard':
+            output_scaled_loss_str = 'z'
+        elif self.loss_scaler_str == 'modified':
+            output_scaled_loss_str = 'modz'
+        else:
+            # in case other custom scaling is used
+            output_scaled_loss_str = f'{self.loss_scaler_str}_scaled' 
+            
         for i, ft in enumerate(self.numeric_fts):
             pdf[ft] = df[ft]
             pdf[ft + '_pred'] = output_df[ft]
             pdf[ft + '_loss'] = mse[:, i].cpu().numpy()
-            pdf[ft + '_z_loss'] = mse_scaled[:, i].cpu().numpy()
+            pdf[ft + f'_{output_scaled_loss_str}_loss'] = mse_scaled[:, i].cpu().numpy()
 
         for i, ft in enumerate(self.binary_fts):
             pdf[ft] = df[ft]
             pdf[ft + '_pred'] = output_df[ft]
             pdf[ft + '_loss'] = bce[:, i].cpu().numpy()
-            pdf[ft + '_z_loss'] = bce_scaled[:, i].cpu().numpy()
+            pdf[ft + f'_{output_scaled_loss_str}_loss'] = bce_scaled[:, i].cpu().numpy()
 
         for i, ft in enumerate(self.categorical_fts):
             pdf[ft] = df[ft]
             pdf[ft + '_pred'] = output_df[ft]
             pdf[ft + '_loss'] = cce[:, i].cpu().numpy()
-            pdf[ft + '_z_loss'] = cce_scaled[:, i].cpu().numpy()
+            pdf[ft + f'_{output_scaled_loss_str}_loss'] = cce_scaled[:, i].cpu().numpy()
```

Comment:
Lets keep the column output names constant and add a new column describing the scaling method used.

Reply:
Done